BUILDING.md

```diff
@@ -188,7 +245,7 @@ The `-j4` option will cause `make` to run 4 simultaneous compilation jobs which
 may reduce build time. For more information, see the
 [GNU Make Documentation](https://www.gnu.org/software/make/manual/html_node/Parallel.html).
 
-Note that the above requires that `python` resolve to Python 2.7 and not a newer
+Note that the above requires that `python` resolve to Python 3 and not a newer
 version.  See [Prerequisites](#prerequisites).
```

Comment:
I would shorten the sentence to `Note that the above requires that 'python' resolve to Python 3.`. The part afterwards should not be required anymore due to the update.

Reply:
‘resolve’ or ‘resolves’?

Reply:
Or even remove this line. `./configure` will warn you anyway. ("tools over rules")

Reply:
What is our minimum Python 3?  Is it >= Python 3.5 or >= Python 3.6?

Reply:
That is a very good question... I test with 3.6 but reading https://devguide.python.org/devcycle/#end-of-life-branches I'll see what's the gap with 3.5.

I think we might want to write add to L220 something like `Python 3 main and development branches, as per the [Python development cylcle](https://devguide.python.org/devcycle/). Specifically 3.6 and 3.7 at the time of release of Node.js 12`

We have a blanket statement that we don't support EOL platforms and software.

Reply:
simpler, I've just gone with:
```
"Note that the above requires that `python` resolves to a supported version of
Python. See [Prerequisites](#prerequisites).
```
we can deal with it in a single place.